wdfn-server/waterdata/views.py

```diff
@@ -21,25 +22,79 @@
 
 NWIS = NwisWebServices(app.config['SERVER_SERVICE_ROOT'], app.config['SITE_SERVICE_CATALOG_ROOT'])
 
-
 @app.route('/')
 def home():
     """Render the home page."""
     return render_template('index.html', version=__version__)
 
 
-@app.route('/questions-comments/<email>/', methods=["GET", "POST"])
-def questions_comments(email):
+def create_message(target_email, form_data, user_system_data, timestamp):
```

Comment:
I think it would be more appropriate to put this in utils.py. It seems pretty general purpose. I'd like view to contain view functions and nothing else (there are always exceptions). Also could you add doc comments?

Reply:
Yes, that makes a lot of sense. Will do, and docs comments added!